src/CMakeLists.txt

```diff
@@ -298,22 +298,46 @@ endif()
 # Installation
 #
 
-include(GNUInstallDirs)
+if(NOT OPENVINO_TOKENIZERS_INSTALL_LIBDIR AND NOT OPENVINO_TOKENIZERS_INSTALL_BINDIR)
```

Comment:
maybe by default we need to fallback top standard cmake CMAKE_INSTALL_BINDIR, CMAKE_INSTALL_LIBDIR ?